blocks/library/heading/index.js

```diff
@@ -108,7 +109,11 @@ registerBlockType( 'core/heading', {
 			),
 			focus && (
 				<InspectorControls key="inspector">
-					<h3>{ __( 'Heading Size' ) }</h3>
+					<BlockDescription>
+						<p>{ __( 'Search engines use the headings to index the structure and content of your web pages.' ) }</p>
```

Comment:
Hah, yes indeed. I think we'll actually want to go over those descriptions altogether, perhaps in a separate PR. The text description is a little jokey as is ;)